internal/server/http/server.go

```diff
@@ -134,9 +135,9 @@ func (s *Server) Shutdown() error {
 	return nil
 }
 
-func (s *Server) addImposterHandler(imposters []Imposter, imposterFilePath string) {
+func (s Server) addImposterHandler(imposters []Imposter, config ImposterConfig) {
```

Comment:
Nitpick: I'd use `imposterConfig` as the variable name, instead. The same way you did for `unmarshalImposters` method.